build-system/tasks/release/index.js

```diff
@@ -0,0 +1,370 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const argv = require('minimist')(process.argv.slice(2));
+const experimentsConfig = require('../../global-configs/experiments-config.json');
+const fetch = require('node-fetch');
+const fs = require('fs-extra');
+const log = require('fancy-log');
+const path = require('path');
+const tar = require('tar');
+const {cyan, green, yellow} = require('ansi-colors');
+const {execOrDie} = require('../../common/exec');
+const {MINIFIED_TARGETS} = require('../helpers');
+const {VERSION} = require('../../compile/internal-version');
+
+// Flavor config for the base flavor type.
+const BASE_FLAVOR_CONFIG = {
+  flavorType: 'base',
+  name: 'base',
+  rtvPrefixes: ['00', '01', '02', '03', '04', '05'],
+  // TODO(#28168, erwinmombay): relace with single `--module --nomodule` command.
+  command: 'gulp dist --noconfig --esm && gulp dist --noconfig',
+  environment: 'AMP',
+};
+
+// Deep map from [environment][flavorType] to RTV prefix.
+const EXPERIMENTAL_RTV_PREFIXES = {
+  AMP: {
+    experimentA: '10',
+    experimentB: '11',
+    experimentC: '12',
+  },
+  INABOX: {
+    experimentA: '20',
+    experimentB: '22',
+    experimentC: '24',
+  },
+};
+
+// Directory containing multiple static files to dist/.
+const STATIC_FILES_DIR = path.resolve(__dirname, 'static');
```

Comment:
I'm glad all this stuff is being moved to our open source repo. In future, it will help keep us honest about what we release, and allow us to remove things that aren't needed.

Reply:
Can you take another look at the actual static files? Are all of them relevant to the open source project, or are some of them specific to the Google servers?

Reply:
I checked to make sure the static files here match up with the ones added internally via cr/229262601. @erwinmombay can you confirm that they are still relevant to our release? (If some of them are Google-only, should they be moved out of `gulp release`?

Reply:
talked to @danielrozenberg offline and agree with his assessment about the static files. when possible we'll keep them external except anything Google only